api/envoy/extensions/request_id/uuid/v3/uuid.proto

```diff
@@ -40,4 +40,8 @@ message UuidRequestIdConfig {
   // stable sampling of traces, access logs, etc. will no longer work and only random sampling will
   // be possible.
   google.protobuf.BoolValue pack_trace_reason = 1;
+
+  // Set whether to use :ref:`x-request-id<config_http_conn_man_headers_x-request-id>` or not.
```

Comment:
```suggestion
  // Set whether to use :ref:`x-request-id<config_http_conn_man_headers_x-request-id>` for sampling or not.
```

Also, please ref link to the section on sampling policy in the arch docs that you added.